scripts/README_merge_truth_per_tract.md

```diff
@@ -0,0 +1,116 @@
+# Using merge_truth_per_tract.py
+
+This README contains the updated instructions for using `merge_truth_per_tract.py`.
```

Comment:
Maybe add a sentence saying what it's for, such as the first sentence from the help, so something like
"This script will merge truth catalogs for a given tract, and then..."